README.md

```diff
@@ -1,141 +1,60 @@
+# Australian Bushfire API and Dashboard
 
 ## Background
 
-This project explores how wildfires have affected the country of Australia over time. Specifically, for this project, we gathered data captured by NASA satellites and various historical wildfire and climate data to help visualize what Australia not only has been through historically but also what Australia has gone through with the most recent (2019-2020) wildfire season. Go to our [website](https://australia-fire-api-dashboard.herokuapp.com/) to learn more about this project.
+This project explores how wildfires have affected Australia over time through data visualizations, including interactive time series charts, plots and a map. Visit our [website](https://australia-fire-api-dashboard.herokuapp.com/) to learn more.
 
-## Loading data into a database
+For this project, we collected open data from a number of sources, including data captured by NASA satellites and various wildfire and climate data. More information about our sources and the data itself can be found in the [Data page](./docs/data.md) in the `docs` folder.
+
+Following, we'll share a background of the project and explain how to recreate the webapp locally, using the code and data shared in this repo.
+
+### Technologies Used
+
+#### Infrastructure
+
+- [Flask](https://flask.palletsprojects.com/en/1.1.x/), a Python micro web framework used to run our API.
+- [MongoDB](https://www.mongodb.com/), a NoSQL for storing the data locally.
+- [MongoDB Atlas](https://www.mongodb.com/cloud/atlas), for storing the data in the cloud.
+- [Heroku](https://www.heroku.com/), for running our dashboard in the cloud.
+- [Jupyter Notebook](https://jupyter.readthedocs.io/en/latest/running.html), for running Python code within a larger document.
+- [Python](https://www.python.org/), including packages like [pandas](https://pandas.pydata.org/) and [Beautiful Soup](https://pypi.org/project/beautifulsoup4/).
+
+#### Data visualization
+
+- [D3.js](https://d3js.org/), a Javascript data visualization library.
+- [d3-timeseries](https://github.com/mcaule/d3-timeseries), a plugin for building time series charts.
+- [Leaflet](https://leafletjs.com/), a Javascript library for building interactive maps.
+
+
+## Installation
+
+### Step 0. Clone the repo
+
+First, you'll need to clone the repository by running:
+
+```git clone git@github.com:philipstubbs13/australia-fire-api-and-dashboard.git```
+
+### Step 1. Load the data
 
 The data for this project is stored in a MongoDB database.
 
-For more information on where this data came from, see the [Data](https://australia-fire-api-dashboard.herokuapp.com/data) page of the site.
+Background information on the [Data](https://australia-fire-api-dashboard.herokuapp.com/data) sources can be found on [our website](https://australia-fire-api-dashboard.herokuapp.com/data), and a detailed list of fields and data types for each database can be found in the [Data page](./docs/data.md) in `docs`.
```

Comment:
typo --> change database to collection:

"and a detailed list of fields and data types for each collection  can be found in..."